libibcm/cm.h

```diff
@@ -325,8 +325,8 @@ struct ib_cm_attr_param {
 int ib_cm_attr_id(struct ib_cm_id *cm_id,
 		  struct ib_cm_attr_param *param);
 
-#define IB_CM_ASSIGN_SERVICE_ID_MASK htobe64(0xFF00000000000000ULL)
-#define IB_CM_ASSIGN_SERVICE_ID      htobe64(0x0200000000000000ULL)
+#define IB_CM_ASSIGN_SERVICE_ID_MASK cpu_to_be64(0xFF00000000000000ULL)
+#define IB_CM_ASSIGN_SERVICE_ID      cpu_to_be64(0x0200000000000000ULL)
 
```

Comment:
I'm really not comfortable with using such common names in public header files.

Is there some sneaky/hacky way you could teach sparse about the glibc macros? Maybe force sparse to include your own endian.h instead of glibc's with the proper annotations?